QuizAPI/DTOs/QuestionDTO.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace QuizAPI.DTOs
+{
+	public class QuestionDTO
+	{
+		private string? question;
+		private string? answer;
+		private string? difficulty;
+		private string? category;
+		private string? status;
+
```

Comment:
We should probably add the tags array here ?

Reply:
The tags weren't pulled through on the scaffold? Didn't notice this

Reply:
The tags were made with the scaffold. This is the DTO, we haven't added the functionality for adjusting a questions tags yet.

Reply:
Ah